scripts/lib/CIME/hist_utils.py

```diff
@@ -455,3 +459,41 @@ def generate_baseline(case, baseline_dir=None, allow_baseline_overwrite=False):
                                                get_timestamp(timestamp_format="%Y-%m-%d_%H:%M:%S")))
 
     return True, comments
+
+def get_ts_synopsis(comments):
```

Comment:
Can you make it clear, through the function name and/or documentation, that this is meant to be called on the comments returned from `compare_baseline`? (What about the comments returned from `compare_test`? My sense is that this would work there, too, but that's not its intended use.)